mender/templates/tests/test-mender.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.tests.enabled }}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "mender.fullname" . }}-test-mender
+  labels:
+    {{- include "mender.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['/bin/sh', '-c']
+      args:
+        {{- if and (.Values.global.enterprise) (.Values.tenantadm.enabled) (.Values.auditlogs.enabled) }}
+        - echo "Starting smoketests";
+          echo "Testing deployment service";
+          wget -qO- '{{ include "mender.fullname" . }}-deployments:{{ .Values.deployments.service.port }}/api/internal/v1/deployments/health';
```

Comment:
Strictly speaking, shouldn't there be a check
```
{{ if .Values.deployments.enabled }}
...
```
Although I'm not sure if it makes much sense running without the services.

Reply:
Figured it was easier to submit a PR than explain it. Here you go: https://github.com/oldgiova/mender-helm/pull/1 :)